### ISSUE TITLE
Staging Review finding: Content references a button that doesn't appear

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#123439]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** MHV Horizon
**Product:** MHV on VA.gov
**Feature:** Notification email update prompts

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content with errors or inconsistencies.](https://design.va.gov/about/experience-standards/#user-encounters-accurate-consistent-content)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @kkitagawa-bosch (content), @jasonday (Accessibility), @allison0034 (Design)

### Description

The "add a contact email" alert references an "add" button, but the button doesn't appear on the page 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="968" height="653" alt="Image" src="https://github.com/user-attachments/assets/973178a2-79e7-4206-a3e7-ff8803bcdcd1" />

</details>

## Recommended action

Add the button 

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Content principles](https://design.va.gov/content-style-guide/content-principles)
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by KKitagawa-Bosch** [comment]
@fmccaf1 We also discussed [on Slack](https://dsva.slack.com/archives/C096YQLRWHK/p1761764095692469?thread_ts=1761758284.491559&cid=C096YQLRWHK), but in the interest of keeping things in one place I'll also comment here: 

I'm really fine with either solution! My concern is just about the content referencing a button that doesn't exist and that potentially being confusing for users. From the content perspective, I think either solution gets the job done about equally as well. 

Our design (@allison0034) or IA (@erinrwhite) leads may have more specific recommendations!
